examples/how-to-use-webrtc-star/index.html

```diff
@@ -0,0 +1,99 @@
+<!doctype html>
```

Comment:
Sorry to be asking this now, but can you make this example a separate PR? I'm not super happy with the explanation provided and I know you are building on top of another contributor work, so don't feel the pressure to change it now. I can improve it :)